src/components/Search/SearchRouter/SearchRouter.tsx

```diff
@@ -76,11 +122,24 @@ function SearchRouter() {
             onClose={closeSearchRouter}
         >
             <FocusTrapForModal active={isSearchRouterDisplayed}>
-                <View style={[styles.flex1, styles.p3]}>
+                <View style={[styles.flex1, styles.p3, modalWidth, styles.mh100, !isSmallScreenWidth && styles.mh85vh]}>
                     <SearchRouterInput
-                        isFullWidth={isFullWidth}
-                        onChange={onSearchChange}
-                        onSubmit={onSearchSubmit}
+                        text={textInputValue}
+                        setText={setTextInputValue}
+                        updateSearch={onSearchChange}
+                        onSubmit={() => {
+                            onSearchSubmit(userSearchQuery);
+                        }}
+                    />
+
+                    <SearchRouterList
```

Comment:
please drop empty line before this, I think in JSX we should not be using empty lines